style.css

```diff
@@ -37,39 +39,34 @@ h1, h2, .me, .info, .projects {
 	width: 250px;
 }
 
-.photos {
+.about-me-photos {
 	display: flex;
 	justify-content: space-evenly;
 	padding: 10px;
+	align-items: flex-end;
 }
 
-.bball {
-	height: 300px;
-	width: 400px;
+.interests {
+	height: auto;
+	max-width: 350px;
 }
 
-.leopard {
-	height: 300px;
-	width: 420px;
+#portfolio {
+	padding: 10px;
 }
 
-
-.box-one, .box-two {
-	width: 95%;
+.box {
+	width: 100%;
 	height: 200px;
```

Comment:
as per my [previous comment](https://github.com/CMeier556/PortfolioProject/pull/8/files/d93068013ff85d8533208695d3ab42fb4e47cd3c..312358913ebd112ccef509515727079bb85e40cd#r1261191369), i think the height here should be removed and allowed to be calculated automatically?